conans/test/tools_test.py

```diff
@@ -74,6 +75,12 @@ def test_environment_nested(self):
         self.assertEquals(os.getenv("B", None), None)
         self.assertEquals(os.getenv("Z", None), None)
 
+    def system_package_tool_fail_when_not_0_returned_test(self):
+        runner = RunnerMock(return_ok=False)
+        spt = SystemPackageTool(runner=runner)
+        with self.assertRaisesRegexp(ConanException, "Command 'sudo apt-get update' failed"):
```

Comment:
This test fails in Win (exception not raised) and in OSX (message is "brew udpate")